Description: Update desktop file
Author: Mateusz Łukasik <mati75@linuxmint.pl>

--- a/obconf.desktop
+++ b/obconf.desktop
@@ -1,5 +1,4 @@
 [Desktop Entry]
-Encoding=UTF-8
 Type=Application
 Name=Openbox Configuration Manager
 Name[pt]=Gestor de configurações do openbox
@@ -17,7 +16,7 @@ Comment[pt_BR]=Configure as opções do
 Comment[ru]=Настройка Openbox
 Comment[zh_TW]=管理 Openbox 組態設定
 Icon=obconf
-Exec=obconf %f
+Exec=obconf
 Categories=Settings;DesktopSettings;GTK;
 MimeType=application/x-openbox-theme;
 StartupNotify=true

